scripts/upload_processor.py

```diff
@@ -0,0 +1,46 @@
+"""A function to process uploaded samples"""
+
+import json
+import google.auth
+from airtable import Airtable
+from google.cloud import secretmanager
+
+
+def get_table(project_config):
+    """Get Table from Airtable"""
+
+    # Get the Airtable credentials.
+    base_key = project_config.get('baseKey')
+    table_name = project_config.get('tableName')
+    api_key = project_config.get('apiKey')
+
+    # Pull table
+    table = Airtable(base_key, table_name, api_key)
+
+    return table
+
+
+def get_config():
+    """Retrieve configuration file from secret manager"""
+
+    # secret_id set for testing, TODO: Replace
+    secret_id = 'test-airtable-config'
+
+    # Get project ID value. First output is a credentials object which isn't used.
+    _, project = google.auth.default()
```

Comment:
I don't think this will work. Keep in mind that this will be executed as part of a Hail Batch pipeline. Therefore the service account that will be available for running this pipeline will be bound to the Hail project, not the dataset project. The dataset / GCP project should probably come as part of the pipeline's configuration for this run -- this is something that @vladsaveliev has probably thought about more.